JS/Lesson17/tugba/script.js

```diff
@@ -134,19 +134,54 @@ console.log('----------------------------');
 // Input: ({name: "John", age: 30}, "city", "New York")
 // Output: {name: "John", age: 30, city: "New York"}
 
+//I'm really not sure about 10-11-12-13 questions.
+const object = { name: "John", age: 30 };
```

Comment:
```suggestion
const object = { name: "John", age: 30 };
object["city"] = "New York";
```
Here I used the bracket notation to add a new key to the object and assign a value to it

Reply:
You don't need the rest of the code except the consol.log statement for this exercise